SingularityService/src/main/java/com/hubspot/singularity/mesos/SingularityMesosStatusUpdateHandler.java

```diff
@@ -308,7 +308,7 @@ private StatusUpdateResult unsafeProcessStatusUpdate(
     long now = System.currentTimeMillis();
     long delta = now - timestamp;
 
-    LOG.debug(
+    LOG.info(
```

Comment:
How much additional logging is this going to be? With the number of tasks running could be overwhelming to have at info level. Same below